app/src/androidTest/java/com/github/wnder/GuessPreviewActivityTest.java

```diff
@@ -48,10 +52,10 @@ public static void setup(){
     @Test
     public void testGuessLocationButton(){
         Intents.init();
-        onView(withId(R.id.guessButton)).perform(click());
 
         // TODO: Check openGuessActivity() correct execution, probably that the activity to make a guess is actually launched and maybe that it sends the image identifier with it
-        // Intents.intended(hasComponent(GuessActivity.class.getName()));
+        //onView(withId(R.id.guessButton)).perform(click());
+        //Intents.intended(hasComponent(GuessLocationActivity.class.getName()));
```

Comment:
Is it still useful?

Reply:
I should have implemented that test so yes it is still to do.
I didn't by lack of time